firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistributionNotificationsManager.java

```diff
@@ -65,24 +65,24 @@ void updateNotification(long totalBytes, long downloadedBytes, UpdateStatus stat
   }
 
   int getNotificationContentTitleId(UpdateStatus status) {
-    if (isErrorState(status)) {
-      return R.string.download_failed;
-    } else if (status.equals(UpdateStatus.DOWNLOADED)) {
-      return R.string.download_completed;
-    } else {
-      return R.string.downloading_app_update;
+    switch (status) {
+      case DOWNLOAD_FAILED:
+        return R.string.download_failed;
+      case INSTALL_FAILED:
+        return R.string.install_failed;
+      case INSTALL_CANCELED:
```

Comment:
The UpdateStatus comes from the ApkUpdater class. We don't currently send Install_canceled or new_release_check_failed. I figured we could keep it for now since those are valid UpdateStatuses. 

In the spots we could be more specific it would require a new ENUM, which does not seem worth it. Ex in apk updater.
https://github.com/firebase/firebase-android-sdk/blob/3072bbcdb5416b0c1accda91a9cd9f24cde6063e/firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/ApkUpdater.java#L240

Overall adding more enums for specificity doesn't seem worth it since we throw firebase exceptions and getting more specific ex install canceled instead of install failed (link to code block below) also would be nice, but not a low lift.
https://github.com/firebase/firebase-android-sdk/blob/3072bbcdb5416b0c1accda91a9cd9f24cde6063e/firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/ApkUpdater.java#L130

Thoughts?

Reply:
Nice investigation. I agree that more specific error messages don't seem worth the effort.

Especially since new ENUMs would require an API change. 

I think that as long as the category of error is correctly displayed, we can point developers and testers to the other SDK logs to debug